var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+import os
+import sys
+from spack import *
+
+
+class Xsdk(Package):
+    """Xsdk is a suite of Department of Energy (DOE) packages for numerical simulation.
+       This is a Spack dummy packages that installs the xSDK packages but 
+       does not itself install anything.
+    """
+
+    homepage = "http://xsdk.info"
+
+    # Dummy url since Spack complains if I don't list something
+    url      = 'https://bitbucket.org/saws/saws/get/master.tar.gz'
+    version('0.1.0','941a541bdf625856be18c9752249146d')
+    # this is for next planned release
+    version('0.2.0','941a541bdf625856be18c9752249146d')
+    # this is for testing with the master of each package 
+    version('master','941a541bdf625856be18c9752249146d')
+
+    # We meed to provide a verions('master' for each package below?
+
+    variant('trilinos', default=True, description='Enables turning off building Trilinos for testing; since it takes so long to build')
+
+    # we actually want 2.11.1 but that isn't available in the Spack file
+    depends_on('hypre@2.10.1~internal-superlu')
+
+    # we actually want git commit 0b5369f but that isn't supported by the Spack file
+    depends_on('superlu-dist@5.0.0')
+
+    depends_on('petsc@3.7.4+mpi+hypre+superlu-dist+metis+hdf5~mumps~boost',when='@0.1.0')
+    depends_on('petsc@master+mpi+hypre+superlu-dist+metis+hdf5~mumps~boost',when='@master')
+
+    depends_on('trilinos@12.6.2+hypre+superlu-dist+metis+hdf5~mumps+boost~suite-sparse',when='+trilinos')
+    #  Looks like someone needs to write a Spack for xsdktrilinos
+    #depends_on('xsdktrilinos',when='+trilinos')
+
+    # We need a way to pass -DUSE_XSDK_DEFAULTS=YES into Trilinos
+    # How do we propagate debug flag to all depends on packages ?
+    #   If I just do spack install xsdk+debug will that propogate it down?
```

Comment:
For this you could just add an `xsdk` variant to Trilinos that adds the option, and depend on `trilinos+xsdk... blahblah`.